PxWeb/wwwroot/Database/EN/EN02/TAB004.px

```diff
@@ -41,9 +41,9 @@ SEASADJ("Substance")=NO;
 UNITS("Substance")="Device varies with the subject";
 DATABASE="Statistical database ";
 SOURCE="Statistics Sweden";
-CONTACT("Substance")="Test Testsson, SCB#Tel: 08-111 222 33#Fax: 08-222 333 44#E-mail: test.testsson@sc"
-"b.se##Test2 Testsson2, SCB#Tel: 08-333 444 55#Fax: 08-444 555 66#E-mail: test2.testsson2"
-"@scb.se";
+CONTACT("Substance")="Test#Testsson##08-111 222 33#test.testsson@scb#SCB##"
+"||Test2#Testsson2##08-111 222 32#test.testsson2@scb#SCB##"
+"||#####just organization##||#####just organization##";
```

Comment:
Where the old px files not valid?

Reply:
They were valid but the contact keyword was using wrong according to how the fileparser works when pipes is used